configure.ac

```diff
@@ -42,6 +42,7 @@ AC_CONFIG_TESTDIR([bootstrap])
 
 AM_INIT_AUTOMAKE([1.9 foreign subdir-objects -Wall])
 
+LT_INIT
```

Comment:
This diff needs to be applied to mute complaints from autotools: https://gist.github.com/ngie-eign/882b78bc57a6b4e50ad224a6394df5e1 . Could you please add this as a [fixup commit](https://mikulskibartosz.name/git-fixup-explained), squash it and other changes I've added to your branch, and rebase with the latest version of master?